unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatHUDController.cs

```diff
@@ -9,19 +9,21 @@ public class ChatHUDController : IDisposable
     public ChatHUDView view;
 
     public UnityAction<string> OnSendMessage;
+    public event UnityAction<string> OnPressPrivateMessage;
 
     public void Initialize(ChatHUDView view = null, UnityAction<string> onSendMessage = null)
     {
-        if (view == null)
-        {
-            this.view = ChatHUDView.Create();
-        }
-        else
-        {
-            this.view = view;
-        }
+        this.view = view ?? ChatHUDView.Create();
 
         this.view.Initialize(this, onSendMessage);
+
+        this.view.OnPressPrivateMessage -= View_OnPressPrivateMessage;
+        this.view.OnPressPrivateMessage += View_OnPressPrivateMessage;
```

Comment:
I think it is always a good practice to unsubscribe all the subscriptions. In this case we could do it in the Dispose method.

Reply:
Done